package.json

```diff
@@ -30,14 +30,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@storybook/addon-actions": "~3.4.12",
-    "@storybook/addon-knobs": "~3.4.12",
-    "@storybook/addon-storysource": "^3.4.12",
-    "@storybook/react": "3.4.10",
-    "@storybook/storybook-deployer": "^2.0.0",
-    "@theforeman/builder": "^3.9.0",
-    "@theforeman/env": "^3.9.0",
-    "@theforeman/vendor-dev": "^3.9.0",
+    "@theforeman/builder": "^3.12.0",
```

Comment:
do we need v4 on @theforeman packages?

Reply:
Already exist here: https://github.com/theforeman/foreman-packaging/pull/4541